src/main/java/com/baidu/hugegraph/driver/HugeClient.java

```diff
@@ -54,53 +48,27 @@
     private AuthManager auth;
     private MetricsManager metrics;
 
-    public HugeClient(String url, String graph) {
-        this(url, graph, DEFAULT_TIMEOUT);
-    }
-
-    public HugeClient(String url, String graph, int timeout) {
-        this(url, graph, timeout, DEFAULT_MAX_CONNS,
-             DEFAULT_MAX_CONNS_PER_ROUTE);
-    }
-
-    public HugeClient(String url, String graph, int timeout,
-                      int maxConns, int maxConnsPerRoute) {
+    public HugeClient create(HugeClientBuilder hugeClientBuilder) {
```

Comment:
just named hugeClientBuilder -> builder is ok, avoid too long

Reply:
use constructor is ok: HugeClient((HugeClientBuilder builder)